src/main/java/org/springframework/social/partnercenter/api/audit/impl/AuditTemplate.java

```diff
@@ -24,13 +25,16 @@ public AuditTemplate(RestResource restResource) {
 
 	@Override
 	public ResponseEntity<PartnerCenterResponse<AuditRecord>> getPartnerCenterActivity(Instant startDate) {
+		notNull(startDate, "[Assertion failed] - startDate argument must be null");
```

Comment:
I think the message should be "... argument must `not` be null"

Reply:
Maybe it would be nice to extract the error message into a constant like `"[Assertion failed] - {} argument must not be null"` and then format the message passing the field name as a parameter.

Reply:
Ha. I copied the message from the default. Guess I should have looked closer. I agree with you suggestion about moving the message template.